coopiteasy_custom/migrations/post-migrate.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Coop IT Easy SC
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    env.cr.execute(
+        """
+    alter table project_task drop column if exists reviewer_id;
+    alter table project_task drop column if exists tester_id;
+    alter table project_task drop column if exists int_priority;
+    drop table link_task_relation_table;
+    """
+    )
+    env.cr.commit()
```

Comment:
this is typically _not done_ in oca projects, preferring to keep (dead) data. here we may not care because we're the only user?

Reply:
yes, i think we can clean it up : these fields were very sparsely used, the data in it is not relevant.